packages/extension-ui/src/components/Mnemonic/MnemonicInput.stories.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { EMPTY_SEED_WORDS, SEED_WORDS_LENGTH } from '@polkadot/extension-ui/Popup/ImportSeed/consts';
+
+import { MnemonicInput } from '.';
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+export default {
+  title: 'Components/Mnemonic/MnemonicInput',
```

Comment:
That's I belive the default title that SB would set for this component, so you don't have to specify it. By default it sets titles based on the location and filename, which imho makes a lot of sense and we should take advantage of that.